src/query/graphite/storage/m3_wrapper_test.go

```diff
@@ -108,11 +108,22 @@ func TestTranslateTimeseries(t *testing.T) {
 		vals := m3ts.NewFixedStepValues(resolution, steps, float64(i), start)
 		series := m3ts.NewSeries([]byte(fmt.Sprint("a", i)),
 			vals, models.NewTags(0, nil))
-		series.SetResolution(resolution)
 		seriesList[i] = series
 	}
 
-	translated, err := translateTimeseries(ctx, seriesList, start, end)
+	resos := make([]int64, 0, expected)
+	for _ = range seriesList {
```

Comment:
nit: You can `for range seriesList { ... }` without needing to `_  = range seriesList`